schemas/symbolicate_api_response_v5.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "http://symbols.mozilla.org/schemas/symbolicate_api_response_v5.json",
+  "description": "Symbolicate API v5 response definition.",
+  "type": "object",
+
+  "definitions": {
+    "job": {
+      "type": "object",
+      "properties": {
+        "found_modules": {
+          "type": "object",
+          "patternProperties": {
+            "[a-zA-Z0-9-\\.]+\/[A-Z0-9]+": {"type": ["boolean", "null"]}
+          }
+        },
+        "stacks": {
+          "description": "list of a list of frame objects",
+          "type": "array",
+          "items": {
+            "type": "array",
+            "items": {
+              "description": "a frame",
+              "type": "object",
+              "properties": {
+                "frame": {"type": "number"},
+                "function": {"type": "string"},
+                "function_offset": {"type": "string"},
+                "module": {"type": "string"},
+                "module_offset": {"type": "string"}
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+
+  "properties": {
+    "results": {
+      "description": "This returns an array of jobs.",
+      "type": "array",
+      "items": {"$ref": "#/definitions/job"}
+    }
+  },
+  "required": [
+    "results"
+  ]
+}
```

Comment:
I'm pretty sure this is correct. It doesn't have the "these properties--and nothing else!" bits, so it's possible it's missing things.

I'll look into that in a follow-up pass.